src/Gui/FreeCADGuiInit.py

```diff
@@ -124,6 +126,27 @@ def GetClassName(self):
         """Return the name of the associated C++ class."""
         return "Gui::NoneWorkbench"
 
+InputSequence = Union[Gui.UserInput, tuple[Gui.UserInput, ...]]

```

Comment:
[PEP 604 – Allow writing union types as X | Y
](https://peps.python.org/pep-0604/):
```suggestion
InputSequence = Gui.UserInput | tuple[Gui.UserInput, ...]
```

Reply:
I have tried to do that but in that context it complained. While it works well in type annotations I think that in case of type alias like this the | operator is ambiguous.